alicloud/resource_alicloud_db_connection.go

```diff
@@ -5,11 +5,18 @@ import (
 	"strings"
 	"time"
 
+	"regexp"
+
 	"github.com/aliyun/alibaba-cloud-sdk-go/services/rds"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
+const dbConnectionSuffixRegex = "\\.mysql\\.([a-zA-Z0-9\\-_]+\\.){0,1}rds\\.aliyuncs\\.com"
```

Comment:
Why is there a char "_" in the dbConnectionSuffixRegex? I think dbConnectionSuffixRegex should be `"\\.mysql\\.([a-zA-Z0-9\\-]+\\.){0,1}rds\\.aliyuncs\\.com"`

Reply:
I just added the '_' just in case because I couldn't find any reference document for this URL. I will remove the '_' then.